src/services/account/services/SupportJWTService.js

```diff
@@ -14,7 +14,7 @@ const logger = require('../../../logger');
 const { getRedisClient, redisSetAsync, extractDataFromJwt, getRedisData } = require('../../../utils/redis');
 
 const DEFAULT_EXPIRED = 60 * 60 * 1000; // in ms => 1h
-const DEFAULT_AUDIENCE = 'https://schul-cloud.org'; // The organisation that create this jwt.
+const DEFAULT_AUDIENCE = 'https://hpi-schul-cloud.org'; // The organisation that create this jwt.
```

Comment:
domain is hpi-schul-cloud.de

Reply:
but docs.hpi-schul-cloud.org has org as ending...i see many mistakes in future if we not move this too.

Reply:
ticketsystem.hpi-schul-cloud.org has also .org

Reply:
hotfix domains has also .org